commit b20cbe55a2aaacd7f33b7bac0df5f497689cd0e4 (refs/changes/20/3020/2)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2017-12-07T10:09:56-08:00 (1 year, 10 months ago)
    
    joyent/manta-thoth#163 Publish install/update updates
    Reviewed by: Bryan Cantrill <bryan@joyent.com>
    Approved by: Bryan Cantrill <bryan@joyent.com>

diff --git a/bin/sdc-thoth-install b/bin/sdc-thoth-install
index 25af95a..d05019c 100755
--- a/bin/sdc-thoth-install
+++ b/bin/sdc-thoth-install
@@ -106,7 +106,7 @@ if [ ! -d $base ]; then
 	tarball=/var/tmp/thoth.$$.tar.gz
 	staged=/tmp/thoth.$$.tar.gz
 	echo "Downloading thoth ..."
-	curl -k $MANTA_URL/thoth/public/thoth.tar.gz > $staged 2>&4
+	curl -k $MANTA_URL/thoth/public/thoth-sunos-latest.tar.gz > $staged 2>&4
 	echo "Downloading thoth to compute nodes ..."
 	sdc-oneachnode -a -g $staged -d /var/tmp 1>&2
 	echo "Installing thoth on compute nodes ..."
diff --git a/bin/sdc-thoth-update b/bin/sdc-thoth-update
index c268c23..65273de 100755
--- a/bin/sdc-thoth-update
+++ b/bin/sdc-thoth-update
@@ -60,7 +60,7 @@ fi
 tarball=/var/tmp/thoth.$$.tar.gz
 staged=/tmp/thoth.$$.tar.gz
 echo "Downloading thoth update ..."
-curl -k $MANTA_URL/thoth/public/thoth.tar.gz > $staged 2>&4
+curl -k $MANTA_URL/thoth/public/thoth-sunos-latest.tar.gz > $staged 2>&4
 echo "Updating thoth ..."
 cd / ; gzcat $staged | tar xf - ; rm $staged 1>&2
 echo "Done"
diff --git a/bin/sdc-thoth-update-all b/bin/sdc-thoth-update-all
index 8e2e547..d337e68 100755
--- a/bin/sdc-thoth-update-all
+++ b/bin/sdc-thoth-update-all
@@ -21,9 +21,9 @@ function onexit
 {
 	if [[ -f $backup ]]; then
 		cp $backup $hosts
-		rm $backup	
+		rm $backup
 	fi
-	
+
 	[[ $1 -ne 0 ]] || exit 0
 	fatal "error exit status $1"
 }
@@ -67,7 +67,7 @@ fi
 tarball=/var/tmp/thoth.$$.tar.gz
 staged=/tmp/thoth.$$.tar.gz
 echo "Downloading thoth ..."
-curl -k $MANTA_URL/thoth/public/thoth.tar.gz > $staged 2>&4
+curl -k $MANTA_URL/thoth/public/thoth-sunos-latest.tar.gz > $staged 2>&4
 echo "Downloading thoth to compute nodes ..."
 sdc-oneachnode -a -g $staged -d /var/tmp 1>&2
 echo "Installing thoth on compute nodes ..."
diff --git a/tools/publish.sh b/tools/publish.sh
index 09770cb..250a14f 100755
--- a/tools/publish.sh
+++ b/tools/publish.sh
@@ -7,4 +7,7 @@ name="thoth-${uname}-${ver:?}-${commit:?}.tar.gz"
 latest="thoth-${uname}-latest.tar.gz"
 
 mput -f "$name" "/thoth/public/$name"
+for c in install update update-all; do
+    mput -f "bin/sdc-thoth-$c" "/thoth/public/sdc-thoth-$c"
+done
 mln "/thoth/public/$name" "/thoth/public/$latest"
